### PR TITLE
feat(): add apply command segment tracking [PHX-2755]

### PR DESCRIPTION
**Pull request by andreascful**
Add Segment tracking for apply command according to tracking plan
- Improve/Update .envrc example

### CODE REVIEW

File: src/commands/apply/index.ts [hunk 1]
```diff
@@ -98,5 +128,9 @@ export default class Apply extends Command {
   protected async finally(): Promise<any> {
     await this.writeFileLog()
     this.log(renderFilePaths({ logFilePath: this.logFilePath }))
+
+    // analyticsCloseAndFlush has a very short timeout because it will
+    // otherwise trigger a rerender of the listr tasks on error exits
+    await Promise.allSettled([Sentry.close(2000), analyticsCloseAndFlush(2000)])
```
Comment (ruderngespra) [inline src/commands/apply/index.ts:134]: The comment is outdated and should be removed (both here and in create/index). This relates to a workaround where we put `1` as param to `analyticsCloseAndFlush`.

**Review by mayakarabula — APPROVED** [review]
lgtm

File: src/analytics/index.ts [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 import crypto from 'crypto'
+import { parseStringAsBoolean } from '../engine/utils/string-to-boolean'
 
 const writeKey = 'YiSkRXCjHUpIDIk9wfbWhoGEGpR99ZXE'
 const userId = crypto.randomUUID()
 
 const analytics = new Analytics({
   writeKey,
-  disable: !!process.env.DISABLE_ANALYTICS,
+  disable: process.env.DISABLE_ANALYTICS ? parseStringAsBoolean(process.env.DISABLE_ANALYTICS) : false,
```
Comment (marcolink) [inline src/analytics/index.ts:11]: what about just checking for `process.env.DISABLE_ANALYTICS === 'true'` here?

Comment (andreascful) [reply src/analytics/index.ts:11]: Or alternatively `process.env.DISABLE_ANALYTICS ? process.env.DISABLE_ANALYTICS.toLowerCase() === 'true' ? false` to cover some of the edge cases of env variable usage.

**Pull request merged**